5_databases/week3/Meal.sql

```diff
@@ -0,0 +1,31 @@
+use homework3;
+--1 Get all meals--
+SELECT*
+from `Meal`;
+--2 ADD new meal --
+INSERT into Meal (`id`,`title`,`description`,`location`,`when`,`max_reservations`,`price`,`created_date`)
+VALUES(1,'couscous','spicy','Tunisia','2023-09-10','10','60','2023-09-20');
+INSERT into Meal (`id`,`title`,`description`,`location`,`when`,`max_reservations`,`price`,`created_date`)
+VALUES(2,'sushi','crispy','japon','2023-04-10','30','50','2023-09-10');
+INSERT into Meal (`id`,`title`,`description`,`location`,`when`,`max_reservations`,`price`,`created_date`)
+VALUES(3,'pizza','tomato','italy','2023-02-10','3','20','2023-09-20');
+INSERT into Meal (`id`,`title`,`description`,`location`,`when`,`max_reservations`,`price`,`created_date`)
+VALUES(4,'Rød grød med','meduim','Denmark','2023-03-10','3','20','2023-09-10');
+--3 Get a meal with any id--
+SELECT*
+from `Meal`
+WHERE id = 1;
+
+--4 update a meal any id --
+UPDATE `Meal`
+SET id = 2
+WHERE id = 1;
+
+--5 update a meal any attribute --
+UPDATE `Meal`
+SET title = 'spaghetti'
+WHERE title = 'couscous';
```

Comment:
You query looks good however, in a real-life scenario always be careful when updating data like this. You have to be certain that you want to update ALL rows with this name, however many there are. If you have 1 row with title `couscous` then all is fine. But you could also have 200 rows with this name.
IF you need to update a specific entry it is generally better to use `WHERE id = ...` as we know there can only be 1 row with that id.

I think you may have misunderstood the task: _Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes_. 

```sql
UPDATE meal
set max_reservations = 8, price = 120
WHERE id = 1;
```